examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
> This seems correct to me - that's why I was thinking either raise a
warning or have an example to illustrate what happens when you apply the
SSP matrix.

Yeah a logger warning could work. If filtering a single channel that is
already part of a projector, warn. (Didn't see your suggestions as I was
composing my email on mobile.) That way it can be verbose='error' to
ignore.

A tutorial example for the linear mixing properties of projection would be
useful.

> Or maybe there could be a check for specific operations done on the data
when apply_proj is called. E.g.:
>
> if raw.has_been_filtered:

Unfortunately we have no backward compatible way of making the *individual
channel* filtering histories persist across write/read which we'd need for
this, so I don't think we can make this very robust. We do save the low and
high pass of the data overall, but that's only if it's applied to all data
channels -- and that's when we don't need to warn.